<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxConnexion">
     <property name="title">
      <string>Connexion</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_IP">
              <property name="text">
               <string>Adresse IP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_IP">
              <property name="text">
               <string>172.18.58.14</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_nomBdd">
              <property name="text">
               <string>Nom de la bdd</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_nomBdd">
              <property name="text">
               <string>banque</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_login">
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_login">
              <property name="text">
               <string>snir</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_mdp">
              <property name="text">
               <string>Mot de passe</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_mdp">
              <property name="text">
               <string>snir</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Connexion à la base de données</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxConnexion_infoComptes">
     <property name="title">
      <string>Affichage infos compte</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_infosCompte">
          <property name="text">
           <string>Afficher les informations de tous les comptes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetInfos">
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>186</number>
          </attribute>
          <column>
           <property name="text">
            <string>Nom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Prénom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Numéro de Compte</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Solde</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonQuitter">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
